virtinst/guest.py

```diff
@@ -1164,6 +1170,8 @@ def _add_spice_sound(self):
         self.add_device(dev)
 
     def _add_spice_usbredir(self):
+        if (self.os.is_loongarch64()):
+            return
```

Comment:
also, why disable this? does libvirt or qemu throw an error? if not IMO we should keep it enabled